src/sinks/prometheus/collector.rs

```diff
@@ -299,6 +299,22 @@ impl StringCollector {
             fullname, name, fullname, r#type
         )
     }
+
+    fn format_tag(key: &str, mut value: &str) -> String {
+        let mut result = String::with_capacity(key.len() + value.len() + 4);
```

Comment:
Nit, it seems like there is only a static extra 3 characters here? `="` and `"`.